recourse_methods/mrmc_method.py

```diff
@@ -121,48 +210,61 @@ class Clusters:
     """Class for kmeans cluster-related data.
 
     Attributes:
-        cluster_assignments: A DataFrame containing columns 'datapoint_index' and 'datapoint_cluster'.
+        cluster_assignments: A DataFrame containing columns 'datapoint_index'
+            and 'datapoint_cluster'.
         cluster_centers: A numpy array containing the cluster centers.
     """
+
     cluster_assignments: pd.DataFrame
     cluster_centers: np.ndarray
 
 
 class MRMC(RecourseMethod):
     """A class for MRM with clustering."""
```

Comment:
In the one-line description, use "Monotonic Recourse Measures (MRM)" for clarity.